tests/unit/drivers/test_flow_visualization.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+import numpy as np
+
+from jina.drivers.helper import array2pb, pb2array
+from jina.flow import Flow
+from jina.proto.jina_pb2 import Document
+from tests import JinaTestCase
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+e1 = np.random.random([7])
+e2 = np.random.random([5])
+e3 = np.random.random([3])
+e4 = np.random.random([9])
+
+
+def input_fn():
+    doc1 = Document()
+    doc1.id = 1
+    doc1.embedding.CopyFrom(array2pb(e1))
+    c = doc1.chunks.add()
+    c.id = 3
+    c.embedding.CopyFrom(array2pb(e2))
+    doc2 = Document()
+    doc2.id = 2
+    doc2.embedding.CopyFrom(array2pb(e3))
+    d = doc2.chunks.add()
+    d.id = 4
+    d.embedding.CopyFrom(array2pb(e4))
+    return [doc1, doc2]
+
+
+class ConcatDriverTestCase(JinaTestCase):
```

Comment:
What is this TestCase?

Reply:
I was using another test case that already had a flow just to check if it was working, I'll make a simple flow one just to visualize it